### PR TITLE
First version of informative README.

### PR DESCRIPTION
**Pull request by Martinsos**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,103 @@
...
+
+# Documentation
+
+While we do plan to have detailed documentation, we don't have anything worth publishing yet due to how fast everything is changing in this early stage of Wasp development.
+
+In the meantime, the best way to learn more is by taking a look at our [TodoApp example](examples/TodoApp).
+
+# Project status
+
+TODO: Describe current progress, where we are, what is the plan. Concisely, in just a couple of sentences, as high-level overview. We want to mention that we are very early and are now doing closed development of , but that generation of that code really works, and that we plan to get waspc to level where it can serve as useful generator, and then get it from that to level where will people will be ejecting later and later. Say which tech stack are we using right now.
```
Comment (Martinsos) [inline README.md]: I have to finish this one before merging!

File: README.md [hunk 2]
```diff
@@ -1,2 +1,112 @@
...
+<p align=center> A programming language that understands what a web app is. </p>
+<br>
+
+[![Gitter](https://badges.gitter.im/wasp-lang/community.svg)](https://gitter.im/wasp-lang/community?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
+
+<br>
+
+Wasp (**W**eb **A**pplication **Sp**ecification Language) is an extensible [DSL](https://en.wikipedia.org/wiki/Domain-specific_language) (domain-specific language) for building modern web apps with less code.
+
+This means that concepts such as *app*, *page*, *user*, *login*, *frontend*, *production* etc. are baked into the language, bringing a new level of expressiveness and allowing you to get a lot of work done with very few lines of code.
```
Comment (matijaSos) [inline README.md]:
```suggestion
This means that concepts such as *app*, *page*, *user*, *login*, *frontend*, *production* etc. are baked into the language, bringing a higher level of expressiveness and allowing you to get more work done with a fewer lines of code.
```

File: README.md [hunk 3]
```diff
@@ -1,2 +1,112 @@
...
+  favicon: "./todo-logo.png"
+}
+
+entity Task {
+  description :: string,
+  isDone :: boolean
+}
+
+entity-list<Task> TaskList {
+  editable: true
```
Comment (matijaSos) [inline README.md]: this we don't have yet, but ok, can easily add it

File: README.md [hunk 4]
```diff
@@ -1,2 +1,112 @@
...
+page Main {
+  route: "/",
+  content: {=jsx <TaskList/> jsx=},
+  style: "./main.css"
+}
+
+...
+```
+
+Source files (.wasp, .js(x), .css, ...) are compiled (transpiled) by `waspc` (Wasp compiler) into the web technology stack of your choice (e.g. React + Redux + ExpressJS + ...).
```
Comment (matijaSos) [inline README.md]:
```suggestion
Source files (`.wasp`, .`js(x)`, `.css`, ...) are compiled (transpiled) by `waspc` (Wasp compiler) into the web technology stack of your choice (e.g. React + Redux + ExpressJS + ...).
```

File: README.md [hunk 5]
```diff
@@ -1,2 +1,112 @@
...
+- **Flexible**: When you need more control than Wasp offers, you can write code in existing technologies such as js/html/css/... and combine it with Wasp code!
+- **No lock-in**: If Wasp becomes too limiting for you, simply eject and continue with the generated source code that follows industry best-practices, as if it was handwritten by a senior engineer.
+
+For more of general information about Wasp, check:
+- **The main webpage**: [https://wasp-lang.dev].
+- [**Blog**](https://blog.wasp-lang.dev/): [Why are we building Wasp?](https://blog.wasp-lang.dev/posts/2019-09-01-hello-wasp.html)
+
+Check out [TodoApp example](examples/todoApp) for complete code example.
+
+If you are interested/excited about Wasp, **support us by giving us a star :star: and/or [subscribe](https://wasp-lang.dev#signup) to our email list**! Check [Contributing](#contributing) for more information.
```
Comment (matijaSos) [inline README.md]:
```suggestion
If you are interested in/excited about Wasp, **support us by giving us a star :star: and/or [subscribe](https://wasp-lang.dev#signup) to our email list**! Check [Contributing](#contributing) for more information.
```

File: README.md [hunk 6]
```diff
@@ -1,2 +1,112 @@
...
+
+
+# This repository
+
+This is the main repo of the Wasp universe, containing core code (not really at the moment, but `waspc` (Wasp compiler) code is coming soon) and supporting materials.
```
Comment (matijaSos) [inline README.md]:
```suggestion
This is the main repo of the Wasp universe, containing core code (not at the moment, but `waspc` (Wasp compiler) code is coming soon) and the supporting materials.
```

File: README.md [hunk 7]
```diff
@@ -1,2 +1,112 @@
...
+
+
+# Documentation
+
+While we do plan to have detailed documentation, we don't have anything worth publishing yet due to how fast everything is changing in this early stage of Wasp development.
```
Comment (matijaSos) [inline README.md]:
```suggestion
While we do plan to publish detailed documentation, we don't have anything worth publishing yet due to how fast everything is changing in this early stage of Wasp development.
```

File: README.md [hunk 8]
```diff
@@ -1,2 +1,112 @@
...
+
+We are still early and are doing closed development iterations with our early adopters.
+
+Specifically, we are developing `waspc` (Wasp compiler) and also designing language as we go.
+
+Currently, `waspc` is in a state where the main parts are there (code analysis and generation) and we are successfully generating code as can be seen at [TodoApp example](examples/todoApp).
+
+Language itself supports a narrow set of features for now, not enough for serious web app development, but it is good basis to build on, we believe.
+The next step is to expand the language (and compiler) so it can serve the role of a powerful code generator - it will still not have enough features to keep you from ejecting for very long time, but it will be enough to give you a quick start.
+Finally, as it grows further, it should allow development of web apps of increasing complexity without the need to eject!
```
Comment (matijaSos) [inline README.md]: nice paragraph

File: README.md [hunk 9]
```diff
@@ -1,2 +1,112 @@
...
+
+While the idea is to support multiple web tech stacks in the future, right now we are focusing on specific stack: React + Redux, NodeJS + ExpressJS, Mongo. We might yet change that as time goes, taking trends into account, but for now this is serving us well for the purpose of developing compiler and language.
```
Comment (matijaSos) [inline README.md]:
```suggestion
While the idea is to support multiple web tech stacks in the future, right now we are focusing on the specific stack: React + Redux, NodeJS + ExpressJS, Mongo. We might yet change that as time goes, taking trends into account, but for now this is serving us well for the purpose of developing compiler and language.
```

File: README.md [hunk 10]
```diff
@@ -1,2 +1,112 @@
...
+
+# Contributing
+
+Since we are still doing closed development of very first version of compiler (`waspc`) and language (check [Project status](#project-status) for more info), you can't contribute to the code directly via this repository at the moment.
+
+However, even more than help with development, at this early stage we need your support and feedback! You can help us greatly by:
+ - :star: Starring this repository.
+ - :mailbox: Staying updated by subscribing to our email list at [wasp-lang.dev](https://wasp-lang.dev#signup).
+ - :speech_balloon: Providing feedback and ideas via gitter [![Gitter](https://badges.gitter.im/wasp-lang/community.svg)](https://gitter.im/wasp-lang/community?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge).
+   Also let us know if you would like to be more closely involved, as we are looking for developers to participate with us in designing / testing the closed version of Wasp before we open it.
```
Comment (matijaSos) [inline README.md]:
```suggestion
   Also let us know if you would like to be more closely involved, as we are looking for developers to participate with us in designing / testing the closed version of Wasp before we open it to the general public.
```

**Pull request merged**